tgbridge/internal/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+type Config struct {
```

Comment:
I think there is no need to separate config for internal if it duplicate the config from cmd.

Reply:
@g0ddest I'm not sure I understand your suggestion here. Can you please elaborate?